docs/src/content/docs/fr/reference/configuration.md

```diff
@@ -424,3 +428,29 @@ starlight({
   ],
 });
 ```
+
+### `titleDelimiter`
+
+**type:** `string`  
+**default:** `'|'`
+
+Définit le délimiteur entre le titre de la page et le titre du site dans la balise `<title>` de la page, qui s'affiche dans les onglets du navigateur.
+
+Par défaut, chaque page a une balise `<title>` contenant `Titre de la page | Titre du site`.
+Par example, cette page a pour titre « Référence de configuration » et ce site a pour titre « Starlight », donc la balise `<title>` de cette page contient « Référence de configuration | Starlight ».
```

Comment:
Reference for the quotation marks: https://www.unicode.org/versions/Unicode10.0.0/ch06.pdf#G19995